global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "3.2.0"
   },
   "sdk": {
-    "version": "6.0.403",
+    "version": "7.0.102",
```

Comment:
Let's hold off with this. This will likely break all our tools that use net 6.0. I was debugging it with @benbp; he had to leave, but I will continue looking into it.

Reply:
OK this might actually work. But let me make a separate PR updating just that.

Reply:
- https://github.com/Azure/azure-sdk-tools/pull/5379

Reply:
Resolved, thanks Konrad~